html/semantics/forms/textfieldselection/selection-not-application.html

```diff
@@ -1,24 +1,27 @@
 <!DOCTYPE html>
 <meta charset=utf-8>
+<meta name=variant content="">
+<meta name=variant content=?week,month">
```

Comment:
An opening quote is needed here, because as it is the closing quote becomes part of the test name:
https://wpt.fyi/results/html/semantics/forms/textfieldselection?diff&filter=ADC&run_id=5724806314983424&run_id=5124632332992512

```suggestion
<meta name=variant content="?week,month">
```